src/main/webapp/index.html

```diff
@@ -5,8 +5,13 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://apis.google.com/js/platform.js" async defer></script>
+  <meta name="google-signin-client_id" content="757099697912-i6jll98mfgochdo2vgjcovf64pepjesc.apps.googleusercontent.com">
```

Comment:
Where is the content URL from?

Reply:
I got that from the gcp dashboard, under the api credentials section

Reply:
I see, can you add a comment for that?

Reply:
added